src/main/java/uk/gov/gchq/palisade/resource/request/AddResourceRequest.java

```diff
@@ -45,8 +46,7 @@ public AddResourceRequest() {
      * @return the {@link AddResourceRequest}
      */
     public AddResourceRequest resource(final LeafResource resource) {
```

Comment:
This needs the @generated annotation on the mutator

Reply:
Done